src/modules/commands/learned.ts

```diff
@@ -0,0 +1,31 @@
+import { Bot } from '../../bot';
+import messages from '../../messages';
+import { Note } from '../../misskey/note';
+import Module from '../../module';
+import { TextProcess } from '../../utils/text-process';
+
+export default class extends Module {
+    Name = 'learned';
+    Regex = /^\/learned (.+)$/i;
+    LogName = 'learned';
```

Comment:
```suggestion
    Name = 'Learned';
    Regex = /^\/learned( \d+)?$/i;
    LogName = 'LRND';
```
LogNameは4文字にしてログを見やすく